xep-0019.xml

```diff
@@ -19,6 +19,12 @@
   <supersededby/>
   <shortname>N/A</shortname>
   &stpeter;
+  <revision>
+    <version>1.1</version>
```

Comment:
Why did you create a revision block for this XEP only? And why did you use a minor level bump instead of a patch level bump?